style/base.css

```diff
@@ -47,7 +49,7 @@
 }
 
 .jp-Dialog-content .jp-Dialog-button {
-  background: linear-gradient(90deg, rgb(0 0 0 / 0%) 0%, rgb(0 0 0 / 20%) 100%), #4b3187 !important;
+  background: var(--je-slate-blue-gradient);
```

Comment:
Does not seem to work anymore as per screenshots? I guess to do with variable definition.